### PR TITLE
Periph oe

### PR DESCRIPTION
**Pull request by gmartin102**
uodates to udma spi module to convert from active high to active low output enables

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,5 @@ build/
 ./ips
 gmon.out
 modelsim.ini
+docs/auto-docs/
```
Comment (zarubaf) [inline .gitignore:11]: Those need to be checked in as RDT isn't capable of producing them. Please remove the ignore, generate the documentation and check it in.

File: docs/auto-docs/apb_gpio_reg_defs.md [hunk 2]
```diff
@@ -1,143 +0,0 @@
-# APB_GPIO
```
Comment (zarubaf) [inline docs/auto-docs/apb_gpio_reg_defs.md:1]: Those files need to be checked in, pls re-add them.

**Review by zarubaf — CHANGES_REQUESTED** [review]
Almost good to go, please add the auto-generated documentation.

**Comment by zarubaf** [comment]
Does this make #119 obsolete or is it a different fix?

**Comment by gmartin102** [comment]
This is an additional fix. But propbab;y also include the irq fix/
Greg


On May 5, 2021, at 3:37 AM, Florian Zaruba ***@***.******@***.***>> wrote:

CAUTION: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.



Does this make #119<https://github.com/openhwgroup/core-v-mcu/pull/119> obsolete or is it a different fix?

**Comment by zarubaf** [comment]
Looks very good to me!

**Review by zarubaf — APPROVED** [review]

**Pull request merged**